### PR TITLE
Improved translation strings

### PR DESCRIPTION
**Pull request by rakekniven**
Improved translation string.
Reported at Transifex. See https://www.transifex.com/nextcloud/nextcloud/translate/#de/$/121491256?issue=yes

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

**Comment by rakekniven** [comment]
Tests are fixed.

File: apps/user_ldap/js/wizard/wizardTabElementary.js [hunk 1]
```diff
@@ -286,7 +286,7 @@ OCA = OCA || {};
 					} else if(objectsFound > 1000) {
 						message = t('user_ldap', 'More than 1,000 directory entries available.');
 					} else {
-						message = t('user_ldap', objectsFound + ' entries available within the provided Base DN');
+						message = t('user_ldap', objectsFound + 'Entries available within the provided Base DN');
```
Comment (rullzer) [inline apps/user_ldap/js/wizard/wizardTabElementary.js]: This looks weird. We probably should just include the number as parameter.

Comment (rakekniven) [reply apps/user_ldap/js/wizard/wizardTabElementary.js]: @rullzer Should I open separate issue regarding overwork of the code?

Comment (rullzer) [reply apps/user_ldap/js/wizard/wizardTabElementary.js]: Nah let me fix it on the branch

Comment (rullzer) [reply apps/user_ldap/js/wizard/wizardTabElementary.js]: Done

**Review by rullzer — APPROVED** [review]

File: apps/user_ldap/js/wizard/wizardTabElementary.js [hunk 2]
```diff
@@ -286,7 +286,9 @@ OCA = OCA || {};
 					} else if(objectsFound > 1000) {
 						message = t('user_ldap', 'More than 1,000 directory entries available.');
 					} else {
-						message = t('user_ldap', objectsFound + ' entries available within the provided Base DN');
+						message = t('user_ldap', '{objectsFound} entries available within the provided Base DN', {
+							objectsFound: objectsFound
```
Comment (MorrisJobke) [inline apps/user_ldap/js/wizard/wizardTabElementary.js]: This should be moved to the plural translation function.

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

**Comment by rakekniven** [comment]
Made a mistake, My last commit (https://github.com/nextcloud/server/pull/8633/commits/1c441e23241f2f0cca80c705ef07f837cb04b5ff) to my branch should not be included in this PR.
Should be separate one.

File: apps/user_ldap/js/wizard/wizardTabElementary.js (hunk 2 above)
Comment (rullzer) [reply apps/user_ldap/js/wizard/wizardTabElementary.js]: fair enough. fixed.

**Comment by rullzer** [comment]
@rakekniven removed your commit

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**